integrationTests/node/index.cjs

```diff
@@ -24,3 +28,20 @@ assert.deepStrictEqual(result, {
     hello: 'world',
   },
 });
+
+const experimentalSchema = buildSchema(`
+  directive @stream(initialCount: Int!) on FIELD
+
+  type Query {
+    greetings: [String]
+  }
+`);
+
+result = experimentalExecuteIncrementally({
+  schema: experimentalSchema,
+  document: parse('{ greetings @stream(initialCount: -1) }'),
+  rootValue: { greetings: ['hi', 'hello'] },
+});
+
+assert(result.errors?.[0] !== undefined);
+assert(!result.errors[0].message.includes('is not defined'));
```

Comment:
can you please add a block comment, explaining what is tested here?

Reply:
Just to clarify, I mean block comments before adding a sub-test.
In the future, if we add more of them it makes sense to have separate files, but it will make `test.js` more complicated.

Reply:
Done! 8700dda